src/db/migrations/45-add-identities.ts

```diff
@@ -0,0 +1,24 @@
+import { Knex } from 'knex';
+
+import { Table } from '../db';
+import { updateViews } from '../migration-helpers';
+
+
+export const up = async (knex: Knex) => {
+
+  await knex.schema.createTable(Table.identities, table => {
```

Comment:
q - should we call this Table.addresses or identities?
 - since address is the primary key, I'd think Table.addresses should be the main
 - we already support multiple addresses on an identity in our other user profile API, and I imagine we'll wanna keep doing this. especially if we wanna converge these 2 databases at some point, I think Table.addresses is closer to that?
 - having said that, name/avatar are more like profile/identity fields rather than address fields

I think for now keeping it as Table.addresses likely makes sense, and just keeping the other fields on this same table for now for simplicity is fine too?

Also maybe it should replace the collectors table too, or we just rename the collectors table to Table.addresses?